selfdrive/camerad/test/check_exposure.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import time
+import unittest
+import os
+import numpy as np
+
+from selfdrive.test.helpers import with_processes
+from selfdrive.camerad.snapshot.snapshot import get_snapshots
+
+from selfdrive.hardware import EON, TICI
+
+WAIT_TIME = 20 # wait for cameras startup and adjustment
+
+os.environ["SEND_ROAD"] = "1"
+os.environ["SEND_DRIVER"] = "1"
+if TICI:
+  os.environ["SEND_WIDE_ROAD"] = "1"
+
+class TestCamerad(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    if not (EON or TICI):
+      raise unittest.SkipTest
+
+  def _numpy_rgb2gray(self, im):
+    ret = np.clip(im[:,:,2] * 0.114 + im[:,:,1] * 0.587 + im[:,:,0] * 0.299, 0, 255).astype(np.uint8)
+    return ret
+
+  def _is_exposure_okay(self, i, med_mean=np.array([[0.2,0.4],[0.2,0.6]])):
+    h, w = i.shape[:2]
+    i = i[h//10:9*h//10,w//10:9*w//10]
+    med_ex, mean_ex = med_mean
+    i = self._numpy_rgb2gray(i)
+    i_median = np.median(i) / 255.
+    i_mean = np.mean(i) / 255.
+    print([i_median, i_mean])
+    return med_ex[0] < i_median < med_ex[1] and mean_ex[0] < i_mean < mean_ex[1]
+
+
+  @with_processes(['camerad'])
+  def test_camera_operation(self):
+    print("checking image outputs")
+
+    time.sleep(WAIT_TIME)
```

Comment:
instead of this fixed sleep, can we just check an image every second until the check passes?